api/types/plugin.go

```diff
@@ -819,6 +820,15 @@ func (s AWSICResourceFilter) MarshalJSON() ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
+func (s *PluginAWSICStatusV1) CheckAndSetDefaults() error {
+	if gis := s.GroupImportStatus; gis != nil {
+		if gis.StatusCode.String() == "" {
+			return trace.BadParameter("invalid group import state %d", gis.StatusCode)
+		}
```

Comment:
I’m not sure why the StatusCode is checked against an empty string.

Could you clarify why this is needed?

AFIK this condition will never be false:  see: https://github.com/gravitational/teleport/blob/master/api/types/types.pb.go#L862 + proto.EnumName